src/server/pkg/storage/fileset/merge.go

```diff
@@ -213,7 +213,7 @@ func (fmr *FileMergeReader) Index() *index.Index {
 }
 
 // Header returns the tar header for the merged file.
-func (fmr *FileMergeReader) Header() (*tar.Header, error) {
+func (fmr *FileMergeReader) Header() (_ *tar.Header, retErr error) {
```

Comment:
Is there a reason to name this return value? It doesn't look like it's referenced. I'm guessing it was at some point in some debugging prints?

Reply:
This has been addressed in a separate PR.